bin/build-mb/resources/overrides.edn

```diff
@@ -27,6 +27,17 @@
                                "jersey-servlet" {:resource "GPL1_1.txt"}}
  "com.sun.jersey.contribs"    {"jersey-guice" {:resource "GPL1_1.txt"}}
  "com.thoughtworks.paranamer" {"paranamer" {:resource "BSD.txt"}}
+ "com.vladsch.flexmark"       {"flexmark"                 {:resource "BSD.txt"}
+                               "flexmark-util-ast"        {:resource "BSD.txt"}
+                               "flexmark-util-builder"    {:resource "BSD.txt"}
+                               "flexmark-util-collection" {:resource "BSD.txt"}
+                               "flexmark-util-data"       {:resource "BSD.txt"}
+                               "flexmark-util-dependency" {:resource "BSD.txt"}
+                               "flexmark-util-format"     {:resource "BSD.txt"}
+                               "flexmark-util-html"       {:resource "BSD.txt"}
+                               "flexmark-util-misc"       {:resource "BSD.txt"}
+                               "flexmark-util-sequence"   {:resource "BSD.txt"}
+                               "flexmark-util-visitor"    {:resource "BSD.txt"}}
```

Comment:
@dpsutton Flexmark seems to use the BSD license, though "BSD" isn't stated anywhere: https://github.com/vsch/flexmark-java/blob/master/LICENSE.txt

It wasn't being picked up automatically so I added these overrides. Let me know if I'm doing anything incorrectly.

Reply:
for something like this i would probably use the overrides/group bit at the top of that file
`"com.vladsch.flexmark" {:resource "BSD.txt"}` instead of each sub project